bcipy/task/paradigm/vep/calibration.py

```diff
@@ -0,0 +1,261 @@
+"""VEP Calibration task-related code"""
+from typing import List, Tuple
+
+from psychopy import core, visual  # type: ignore
+
+from bcipy.acquisition.multimodal import ClientManager
+from bcipy.config import TRIGGER_FILENAME, WAIT_SCREEN_MESSAGE
+from bcipy.display import InformationProperties, VEPStimuliProperties
+from bcipy.display.components.layout import centered
+from bcipy.display.components.task_bar import CalibrationTaskBar
+from bcipy.display.paradigm.vep.codes import round_refresh_rate, ssvep_to_code
+from bcipy.display.paradigm.vep.display import VEPDisplay
+from bcipy.display.paradigm.vep.layout import BoxConfiguration
+from bcipy.helpers.clock import Clock
+from bcipy.helpers.parameters import Parameters
+from bcipy.helpers.symbols import alphabet
+from bcipy.helpers.task import get_user_input, trial_complete_message
+from bcipy.helpers.triggers import (FlushFrequency, Trigger, TriggerHandler,
+                                    TriggerType, convert_timing_triggers,
+                                    offset_label)
+from bcipy.task import Task
+from bcipy.task.paradigm.vep.stim_generation import (
+    InquirySchedule, generate_vep_calibration_inquiries)
+
+
+def trigger_type(symbol: str, target: str, _index: int) -> TriggerType:
+    """Trigger Type.
+
+    This method is passed to convert_timing_triggers to properly assign TriggerTypes
+        to the timing of stimuli presented.
+    """
+    if target == symbol:
+        return TriggerType.TARGET
+    return TriggerType.EVENT
```

Comment:
We may need to support different TriggerTypes for this. A good question for the group at demo. What triggers will be needed for reshaping / future analysis. I added a few comments in the display that I know we will want.